spec/index.bs

```diff
@@ -4328,15 +4330,15 @@ dictionary GPUExternalTextureDescriptor : GPUObjectDescriptorBase {
 
             1. Let |source| be |descriptor|.{{GPUExternalTextureDescriptor/source}}.
 
-            1. Let |usability| be the result of
-                [=check the usability of the image argument|checking the usability of=] |source|
-                (which may throw an exception).
-
-            1. If |usability| is `bad`, throw an {{InvalidStateError}} and stop.
-
             1. If |source| <l spec=html>[=is not origin-clean=]</l>,
                 throw a {{SecurityError}} and stop.
 
+            1. Let |usability| be [=?=] [=check the usability of the image argument=](|source|).
+
+            1. If |usability| is not `good`:
+                1. [$Generate a validation error$].
+                1. Return.
```

Comment:
Out-of-style for now because we have no asynchronous validation block here right now. But when we add one we'll put it in the right place.